server/monitor/src/main/appended-resources/META-INF/LICENSE

```diff
@@ -57,9 +57,21 @@ Files:
 
     Flot bundles additional works:
 
-    jquery.pie.js
-    * Created by Brian Medendorp, June 2009
-    * Updated November 2009 with contributions from: btburnett3, Anthony Aragues and Xavi Ivars
+    jquery.flot.pie.js
+        Flot plugin for rendering pie charts.
+
+        Copyright (c) 2007-2014 IOLA and Ole Laursen.
+        Licensed under the MIT license.
+
+        The plugin assumes that each series has a single data value, and that each
+        value is a positive integer or zero.  Negative numbers don't make sense for a
+        pie chart, and have unpredictable results.  The values do NOT need to be
+        passed in as percentages; the plugin will calculate the total and per-slice
+        percentages internally.
+
+        * Created by Brian Medendorp
+
```

Comment:
Can remove this blank line (same for the above copies) between these attributions.